### PR TITLE
New accordion component and created a new component called expandable-tile

### PR DESCRIPTION
**Pull request by OsamaAtReloadly**

### CODE REVIEW

**Review by salimzakari-reloadly — APPROVED** [review]

File: src/lib/components/expandable-tile/expandable-tile.component.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  templateUrl: './expandable-tile.component.html',
+  styleUrls: ['./expandable-tile.component.scss']
+})
+export class ReloadlyExpandableTileComponent {
+    @Input() title?: string;
+    @Input() disabled: boolean = false;
+    @Input() width: string = '176px'
+    @Output() onToggle: EventEmitter<boolean> = new EventEmitter();
+
+
```
Comment (SabreFoxx-Reloadly) [inline src/lib/components/expandable-tile/expandable-tile.component.ts:14]: Extra spacing

File: src/lib/components/expandable-tile/expandable-tile.module.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { ReloadlyExpandableTileComponent } from './expandable-tile.component';
+import { NgModule } from '@angular/core';
+
+
```
Comment (SabreFoxx-Reloadly) [inline src/lib/components/expandable-tile/expandable-tile.module.ts:4]: Extra spacing

File: src/lib/components/expandable-tile/expandable-tile.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (SabreFoxx-Reloadly) [inline src/lib/components/expandable-tile/expandable-tile.component.spec.ts:2]: Extra spacing

File: src/lib/components/expandable-tile/expandable-tile.component.scss [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+            .content {
+                padding: 4px;
+            }
+        }
+    }
+
+    .expandable-tile-header {
+        display: flex;
+
```
Comment (SabreFoxx-Reloadly) [inline src/lib/components/expandable-tile/expandable-tile.component.scss:83]: Extra spacing

**Review by iSeremet-Reloadly — APPROVED** [review]

File: src/lib/components/expandable-tile/expandable-tile.component.scss [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+            width: 10px;
+            height: 10px;
+            border-bottom: 2px solid;
+            border-right: 2px solid;
+            transform: rotate(45deg);
+            left: 4px;
+            top: 2px
+        }
+
+
```
Comment (SabreFoxx-Reloadly) [inline src/lib/components/expandable-tile/expandable-tile.component.scss:53]: Extra spacing

File: src/lib/components/accordion/accordion.module.ts [hunk 6]
```diff
@@ -1,11 +1,29 @@
...
-    providers: [],
+  imports: [
+    CommonModule,
+    BrowserModule,
+    BrowserAnimationsModule],
+  exports: [
+    ReloadlyAccordionComponent,
+    ReloadlyAccordionItem,
+    ReloadlyAccordionItemContent,
+
```
Comment (SabreFoxx-Reloadly) [inline src/lib/components/accordion/accordion.module.ts:20]: Extra spacing

**Review by SabreFoxx-Reloadly — APPROVED** [review]

**Review by iSeremet-Reloadly — APPROVED** [review]

**Pull request merged**